backend/data/annotations/commentaries/text_annotations_tlg0003.tlg001.perseus-grc2.json

```diff
@@ -0,0 +1,160 @@
+[
```

Comment:
@jtauber this fixture was created manually from https://github.com/pletcher/canonical_pdlrefwk/blob/6ed59ffa60dda99062ce3f8a712b445f9cdc4554/data/viaf76387703/viaf001/viaf76387703.viaf001.perseus-eng1.xml#L67.

- `references` is to the CTS passage
- `commentary` supports HTML styling and is content extracted from `viaf76387703.viaf001.perseus-eng1.xml`
- `fragment` is what is shown in the widget sidebar (roughly `lem`)
- `ve_refs` are the token references in the passage (e.g. `urn:cts:greekLit:tlg0003.tlg001.perseus-grc2:1.1.1@ Θουκυδίδης` == `1.1.1.t1`